shared/message/vision/Obstacle.proto

```diff
@@ -30,11 +30,12 @@ message Obstacle {
         MAGENTA      = 1;
         CYAN         = 2;
     }
-
-    uint32                    id        = 1;
+    uint32 id = 1;
+    /// When the obstacle was seen
     google.protobuf.Timestamp timestamp = 2;
-    Polygon                   shape     = 3;
-    Team                      team      = 4;
+    /// The {x, y} point location of the obstacle
```

Comment:
This seems wrong

Reply:
changed to `{x, y} points bounding the location of the obstacle`